package.json

```diff
@@ -35,7 +35,7 @@
     "convert-source-map": "^1.3.0",
     "error-stack-parser": "^1.3.6",
     "eslint": "^3.8.1",
-    "lab": "^11.1.0",
+    "lab": "^13.1.0",
```

Comment:
Ahhh, I see, yeah looking at the total diff would make it seem arbitrary indeed.